scripts/variantstore/beta_docs/gvs-quickstart.md

```diff
@@ -27,7 +27,7 @@ While the GVS workflow has been tested with 100,000 single sample GVCF files as
 
 The following files are stored in the workspace Google bucket and links to the files are written to the `sample_set` data table:
 
-- Sharded joint VCF files and index files
+- Sharded joint VCF files, index files, the interval lists produced for this run, and the sample name list
```

Comment:
```suggestion
- Sharded joint VCF files, index files, the interval lists for each sharded VCF, and a list of the sample names included in the callset.
```

Thoughts?

Reply:
I'm fine with that verbiage, and in fact think it's better.  I can change it in gvs-quickstart.md and run-your-own-samples.md, as it's repeated in both places

Reply:
Sounds good, thanks